presto-main/src/main/java/com/facebook/presto/server/InternalCommunicationConfig.java

```diff
@@ -15,13 +15,20 @@
 
 import io.airlift.configuration.Config;
 
+import java.io.File;
+
 public class InternalCommunicationConfig
 {
     private boolean httpsRequired;
     private String keyStorePath;
     private String keyStorePassword;
     private String ldapUser;
     private String ldapPassword;
+    private String kerberosPrincipal;
+    private String kerberosServiceName;
+    private boolean kerberosEnabled;
+    private File kerberosKeytab;
+    private File kerberosConfig;
```

Comment:
You need 2 more properties here. `kerberosCredentialCache` and `kerberosUseCanonicalHostname`.